### DISCUSSION TITLE
How to disable emoji's when I type :

### DISCUSSION THREAD

**Discussion by aindriu80** [LazyVim/LazyVim#4952] [Q&A] [answered]
Does anyone know how to disable the emoji pop up when you type colon : 

here is a screenshot 

![2024-12-04_17-28](https://github.com/user-attachments/assets/b986d43e-0dad-419f-aa07-b36703c12478)

I don't use emoji coding, i did search, thanks sooo much

**Comment by folke** [comment, marked as answer]
That can only show up if you added `cmp-emoji` in  your config
